velox/exec/fuzzer/DuckQueryRunner.cpp

```diff
@@ -386,24 +433,27 @@ std::optional<std::string> DuckQueryRunner::toSql(
 
   switch (joinNode->joinType()) {
     case core::JoinType::kInner:
-      sql << " FROM t INNER JOIN u ON " << joinConditionAsSql(joinNode);
+      sql << " FROM " << probeTableName << " INNER JOIN " << buildTableName
+          << " ON " << joinConditionAsSql(*joinNode);
       break;
     case core::JoinType::kLeft:
-      sql << " FROM t LEFT JOIN u ON " << joinConditionAsSql(joinNode);
+      sql << " FROM " << probeTableName << " LEFT JOIN " << buildTableName
+          << " ON " << joinConditionAsSql(*joinNode);
       break;
     case core::JoinType::kFull:
-      sql << " FROM t FULL OUTER JOIN u ON " << joinConditionAsSql(joinNode);
+      sql << " FROM " << probeTableName << " FULL OUTER JOIN " << buildTableName
+          << " ON " << joinConditionAsSql(*joinNode);
       break;
     case core::JoinType::kLeftSemiFilter:
       // Multiple columns returned by a scalar subquery is not supported in
-      // DuckDB. A scalar subquery expression is a subquery that returns one
+      // Presto. A scalar subquery expression is a subquery that returns one
```

Comment:
nit, this should be true for both ? or is this false for DuckDb ?

Reply:
This is true for both Presto and DuckDB